proxies/shared/partials/Partial.Target.FaultRules.xml

```diff
@@ -6,6 +6,24 @@
             response.content ~~ "\{\u0022message\u0022:\u0022.*\u0022\}"
         </Condition>
     </Step>
+    <Step>
```

Comment:
Not too clued up on APIGEE side of things, but do these steps get applied to all errors returned from the API irregardless of the endpoint?

Reply:
Correct, I personally don't think we should try and restrict these new policies for a given set of endpoints. Keeping those errors generic might be helpful in the future.

Reply:
Are they applied sequentially? I'm just wondering if there's any issues with applying the 1st step to a none NhsAppAccounts endpoint error? (I realise this is in prod already and I'm lacking a lot of knowledge in this area)

Reply:
Yes they apply sequentially. I first step extract the message from the response from the BE and puts it in a context variable. It's a bit of a mess at the moment, it was added as part of the NHS App Accounts work but I don't think that policy should have "NhsAppAccounts" in its name.

Reply:
Oh ok, I mean if you're happy to, could rename it to something more generic?

Reply:
Done :)